digitalstrategie/assets/scss/components/_ds-block.scss

```diff
@@ -27,7 +27,6 @@
         h3,
         h3:after,
         h4 {
-            background-color: $green !important;
```

Comment:
Ah! The background comes from the other (colour-classes) and that is fine? Cool!

Reply:
yes it already comes from the parent class